src/main/java/org/wildfly/security/auth/realm/ldap/OtpCredentialLoader.java

```diff
@@ -128,8 +129,8 @@ public SupportLevel getCredentialAcquireSupport(final Class<? extends Credential
                 Password password = passwordFactory.generatePassword(new OneTimePasswordSpec(
                                 CodePointIterator.ofString((String) hashAttribute.get())
                                         .base64Decode(Alphabet.Base64Alphabet.STANDARD, false).drain(),
-                                CodePointIterator.ofString((String) seedAttribute.get())
-                                        .base64Decode(Alphabet.Base64Alphabet.STANDARD, false).drain(),
```

Comment:
Here there is something that needs an additional review, probably from @honza889 :

In FileSystemSecurityRealm:690 we are storing into the filesystem the current seed value with: `streamWriter.writeAttribute("seed", ByteIterator.ofBytes(otp.getSeed()).base64Encode().drainToString());`
In FileSystemSecurityRealm:968 we are retrieving it with: `seed = CodePointIterator.ofString(streamReader.getAttributeValue(i)).base64Decode().drain();`

We are doing the same when the seed is being stored in a LDAP server but with a small difference:
In OtpCredentialLoader:157 we are storing the seed with: `attributes.put(seedAttributeName, ByteIterator.ofBytes(password.getSeed()).base64Encode().drainToString());`
In OtpCredentialLoader:131 we are retrieving the seed with: `CodePointIterator.ofString((String) seedAttribute.get()).base64Decode(Alphabet.Base64Alphabet.STANDARD, false).drain()`

I have discussed it with @fjuma , but currently we are not sure if it was intentional to read the seed from LDAP using requirePadding=false or not. If it was intentional, shouldn't we do the same in FileSystemSecurityRealm?. If it wasn't intentional, maybe this can be removed because the seed for LDAP is just being stored using base64Encode(), which always adds trailing padding characters when necessary.